code/utils.py

```diff
@@ -20,11 +23,8 @@ def get_classes(folder: Path) -> List[str]:
     Returns:
         A list of strings corresponding to the class names.
     """
-    return sorted([
-        f.name for f in folder.iterdir()
-        if ((folder.joinpath(f.name).is_dir()) and (".DS_Store" not in f.name))
-    ],
-                  key=str)
+    return sorted([f.name for f in folder.iterdir() if
+                   ((folder.joinpath(f.name).is_dir()) and (".DS_Store" not in f.name) and (f.suffix.casefold() in IMAGE_EXTS))], key=str)
```

Comment:
as .is_dir() suggests, this function is about folders, so no need to check extension.